File: BeeCrowd/Python/beecrowd_1188.py
add, count = 0 , 0

tag = input()

for i in range(12):
    for j in range(12):
        x = float(input())
        if(i>6 and j>11-i and i>j):
            add += x
            count += 1

print("%.1f" %(add if tag == 'S' else add/count))

File: BeeCrowd/Cpp/beecrowd_1188.cpp
#include <stdio.h>

int main() {

  char tag;
  double n, sum = 0.0, count = 0.0;
  int i, j;

  scanf("%c", &tag);

  for (i = 0; i < 12; ++i) {
    for (j = 0; j < 12; ++j) {
      scanf("%lf", &n);
      if (i > 6 && j > 11 - i && i > j) {
        sum += n;
        count += 1;
      }
    }
  }

  printf("%.1lf\n", (tag == 'S') ? sum : sum / count);
  return 0;
}
